code/datums/mind.dm

```diff
@@ -1253,43 +1252,37 @@
 					log_admin("[key_name(usr)] has unemag'ed [ai]'s Cyborgs.")
 
 	else if (href_list["handofgod"])
-		switch(href_list["handofgod"])
-			if("clear") //wipe handofgod status
-				if((src in ticker.mode.red_deity_followers) || (src in ticker.mode.blue_deity_followers) || (src in ticker.mode.red_deity_prophets) || (src in ticker.mode.blue_deity_prophets))
-					remove_hog_follower_prophet()
-					current << "<span class='danger'><B>You have been brainwashed... again! Your faith is no more!</B></span>"
-					message_admins("[key_name_admin(usr)] has de-hand of god'ed [current].")
-					log_admin("[key_name(usr)] has de-hand of god'ed [current].")
-
-			if("red follower")
-				make_Handofgod_follower("red")
-				message_admins("[key_name_admin(usr)] has red follower'ed [current].")
-				log_admin("[key_name(usr)] has red follower'ed [current].")
-
-			if("red prophet")
-				make_Handofgod_prophet("red")
-				message_admins("[key_name_admin(usr)] has red prophet'ed [current].")
-				log_admin("[key_name(usr)] has red prophet'ed [current].")
-
-			if("blue follower")
-				make_Handofgod_follower("blue")
-				message_admins("[key_name_admin(usr)] has blue follower'ed [current].")
-				log_admin("[key_name(usr)] has blue follower'ed [current].")
-
-			if("blue prophet")
-				make_Handofgod_prophet("blue")
-				message_admins("[key_name_admin(usr)] has blue prophet'ed [current].")
-				log_admin("[key_name(usr)] has blue prophet'ed [current].")
-
-			if("red god")
-				make_Handofgod_god("red")
-				message_admins("[key_name_admin(usr)] has red god'ed [current].")
-				log_admin("[key_name(usr)] has red god'ed [current].")
-
-			if("blue god")
-				make_Handofgod_god("blue")
-				message_admins("[key_name_admin(usr)] has blue god'ed [current].")
-				log_admin("[key_name(usr)] has blue god'ed [current].")
+		if(href_list["handofgod"] == "clear") //wipe handofgod status
+			ticker.mode.remove_hog_follower(src, 0)
+			current << "<span class='danger'><B>You have been brainwashed... again! Your faith is no more!</B></span>"
+			message_admins("[key_name_admin(usr)] has de-hand of god'ed [current].")
+			log_admin("[key_name(usr)] has de-hand of god'ed [current].")
+
+		else if(href_list["handofgod"] in teams)
+			make_Handofgod_follower(href_list["handofgod"])
+			message_admins("[key_name_admin(usr)] has red follower'ed [current].")
+			log_admin("[key_name(usr)] has red follower'ed [current].")
+
```

Comment:
Might get confused with ''Admin has followed user'' from when we orbit and stuff.